src/App.test.js

```diff
@@ -21,6 +19,6 @@ test("Typeracer renders after starting game", () => {
     userEvent.click(startGameButton);
     // Verify that the typeracer UI is displayed after clicking "Start Game"
     expect(screen.getByText("Begin typing:")).toBeInTheDocument();
-    expect(document.querySelector(".prompt-text")).toBeInTheDocument();
-    expect(document.querySelector(".player-input")).toBeInTheDocument();
+    expect(document.querySelector(".p")).toBeInTheDocument();
+    expect(document.querySelector(".bXYnqS")).toBeInTheDocument();
```

Comment:
It isn't clear what these 2 lines are testing, I would use the same names from before or something similar.

Reply:
those names are default names from styling